src/Sign.Cli/Sign.Cli.csproj

```diff
@@ -67,6 +67,10 @@
       <Pack>true</Pack>
       <PackagePath>tools\$(TargetFramework)\any\tools</PackagePath>
     </Content>
+    <Content Include="$(RepositoryRootDirectory)\THIRD-PARTY-NOTICES.txt">
```

Comment:
Does this file exist yet?

Reply:
Yes, in the [cli](https://github.com/dotnet/sign/tree/cli) branch, which is not the default branch.